.github/workflows/ci-vulnerability-scans.yml

```diff
@@ -53,13 +61,22 @@ jobs:
       - name: Run Trivy vulnerability scan
         uses: aquasecurity/trivy-action@master
         with:
-          scan-type: "image" # Scan type, image or fs
+          scan-type: image
           image-ref: ${{ steps.build-image.outputs.image }}
-          format: "table" # Can be table, json, sarif, or github
-          exit-code: "1" # Exit code to return if any vulnerabilities are found
-          ignore-unfixed: true # Ignore unpatched/unfixed vulnerabilities
-          vuln-type: "os" # Can be os or library, depending on the type of scan you want to run
-          security-checks: "vuln,secret" # Can be any combo of vuln, secret, and config
+          format: table
+          exit-code: 1
+          ignore-unfixed: true
+          vuln-type: os,library
```

Comment:
Could we leave it as `os` for this PR since this PR is just about changing formatting? i'm slightly worried that the extra scope of scanning might cause this workflow to fail on the platform-test, platform-test-nextjs, or platform-test-flask repos. but i'm also ok with yolo'ing it if you want and just fixing it after the fact.

Reply:
I had that locally, but forgot to push it, in this next commit